File: scala-akka-http-server/src/main/scala/org/openapitools/server/model/UserGroup.scala
package org.openapitools.server.model


/**
 * Object containing the user group's attributes. 
 *
 * @param name The name of the user group.  for example: ''null''
 * @param description The description of the user group.  for example: ''null''
 * @param members Array containing the id of the users who are members of this user group.  for example: ''null''
 * @param id The ID of the user group.  for example: ''null''
*/
final case class UserGroup (
  name: Option[String],
  description: Option[String],
  members: Option[Seq[Int]],
  id: Option[Int]
)

File: kotlin-spring/src/main/kotlin/org/openapitools/model/UserGroup.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size
import javax.validation.Valid

/**
 * Object containing the user group's attributes. 
 * @param name The name of the user group. 
 * @param description The description of the user group. 
 * @param members Array containing the id of the users who are members of this user group. 
 * @param id The ID of the user group. 
 */
data class UserGroup(

    @field:JsonProperty("name") val name: kotlin.String? = null,

    @field:JsonProperty("description") val description: kotlin.String? = null,

    @field:JsonProperty("members") val members: kotlin.collections.List<kotlin.Int>? = null,

    @field:JsonProperty("id") val id: kotlin.Int? = null
) {

}

